de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java

```diff
@@ -832,8 +843,10 @@ private static FskPortObject getFSKObjectFromStringArray(FskPortObject portObjec
       String modelType) throws JsonMappingException, JsonProcessingException, IOException {
 
     portObject.modelMetadata = MAPPER.readValue(model[0], SwaggerUtil.modelClasses.get(modelType));
-    portObject.setModel(MAPPER.readValue(model[1], String.class));
-    portObject.setViz(MAPPER.readValue(model[2], String.class));
+    if(!StringUtils.isEmpty(model[1]))
```

Comment:
Since we are using StringUtils we could use isNotEmpty instead of negating the conditions.